### PR TITLE
docs: Reorder sections, add first badge

### PR DESCRIPTION
**Pull request by karfau**
to show usage (incl. input and output) earlier in the readme

### CODE REVIEW

**Review by StickFigureJakob — COMMENTED** [review]
What do you think about this order?:
Scor
Vision
Usage
Concept
TODOs ?

File: README.md [hunk 1]
```diff
@@ -111,16 +61,71 @@ const scorsList = [
...
+The different relevant values come in very different "shapes". Once all the data
+is gathered per package, depending on the use case the different values are more
+or less relevant.
+
+I experienced that such a "rating system", or "weighted average score", is not
+so easy to get completely right from scratch alongside collecting the data. It
+also involves a lot of repetitive code that easily leaks its abstractions into
+the rest of the code.
+
+`scor` simplifies this by making certain assumptions:
```
Comment (StickFigureJakob) [inline README.md:102]: Maybe deviding this chapter here would help to make it easier to read

**Comment by karfau** [comment]
@StickFigureJakob I moved the "use case" part to the usage section, before the code example.
Thx for the suggestion.

Does it make more sense for you now?

**Review by StickFigureJakob — APPROVED** [review]
much better :100:

**Pull request merged**